helm/ngrok-operator/tests/controller-deployment_test.yaml

```diff
@@ -222,19 +222,6 @@ tests:
   - contains:
       path: spec.template.spec.containers[0].args
       content: --zap-log-level=error
-- it: Should pass bindings allowed urls if set
```

Comment:
nit: probably just change this test for endpoint selectors to make sure the containers args still get passed and we're templating correctly